packages/core/src/components/fl-badge/test/fl-badge.spec.tsx

```diff
@@ -0,0 +1,22 @@
+import { newSpecPage } from "@stencil/core/testing";
+import { FlBadge } from "../fl-badge";
+// Gonna ignore the comment about double quotes
+// Single quotes need love too ;)
+/*eslint quotes: ["error", "double", { "allowTemplateLiterals": true }]*/
+describe("fl-badge", () => {
+  it("renders", async () => {
+    const page = await newSpecPage({
+      components: [FlBadge],
+      html: `<fl-badge></fl-badge>`,
+    });
+    expect(page.root).toEqualHtml(`
+      <fl-badge class="badge badge-- badge--primary">
+        <mock:shadow-root>
+	  primary
```

Comment:
Does this text serve any purpose?

Reply:
No, i believe that's some default text but I'll remove it.

Reply:
Oh, actually that was used to pass the linter since the component used default names originally.